src/components/forms/password-input.jsx

```diff
@@ -34,6 +34,7 @@ const PasswordInput = React.forwardRef(({ className, ...props }, ref) => {
       </Button>
 
       {/* hides browsers password toggles */}
+      {/* TODO: These prefixes are IE specific and probably could be dropped */}
```

Comment:
@alvarobernal2412 Do you recall why you added this? Would be great to remove it in this PR as well.

Reply:
He told me privately that it's possible to remove them